common-lib/project.clj

```diff
@@ -41,17 +41,17 @@
                  [org.clojure/tools.nrepl "0.2.13"]
                  [org.clojure/tools.reader "1.3.2"]
                  ;;these dependencies should be updated in tandem with the ring dependiences below
```

Comment:
I actually think that we want to have the ` 9.4.51.v20230217` version on here. I think this from going to https://clojars.org/ring/ring-jetty-adapter/versions/1.10.0 and then the dependency on the right side is that value. Either way we should add a comment here that explains how to find the matching dependency that existing comment is not very helpful.

Reply:
The version I upgraded to is `1.9.6`, not `1.10.0`, which depends on `9.4.48.v20220622`

Reply:
You are correct should have looked more closely I just went to latest ty!